index.html

```diff
@@ -170,6 +170,11 @@ <h2 id="project_tagline"></h2>
 <pre><code> $ java -jar publish-cli.jar -f publishMessages.json -en rabbit -mb mb123 -np -mp eiffelsemantics
 </code></pre>
 
+<p><strong>For loading protocol jars other than eiffelsemantics :</strong></p>
+
+<pre><code> $ java -Djava.ext.dirs="Path to eiffel3messaging protocol jars" -jar publish-cli.jar -f publishMessages.json -mp eiffel3
+</code></pre>
```

Comment:
Please provide a simple example  on where the protocol jars reside and how we need to declare it, so that it will be very understandable for the end users